docs-source/content/userguide/managing-domains/accessing-the-domain/admin-console.md

```diff
@@ -1,6 +1,84 @@
 ---
-title: "Administration Console"
+title: "Use the Remote Console"
 date: 2019-02-23T17:39:15-05:00
-draft: true
+draft: false
 weight: 2
+description: "Use the Oracle WebLogic Server Remote Console to manage a domain running in Kubernetes."
 ---
+
+The Oracle WebLogic Server Remote Console is a lightweight, open source console that does not need to be collocated with a WebLogic Server domain.
+You can install and run the Remote Console anywhere. For an introduction, read the blog, "The NEW WebLogic Server Remote Console."
```

Comment:
the link to the blog https://blogs.oracle.com/weblogicserver/new-weblogic-server-remote-console